db-engine/src/main/java/com/splicemachine/db/iapi/sql/compile/OptimizerFlag.java

```diff
@@ -105,7 +105,9 @@
     COST_INCLUDING_STATS_FOR_INDEX(TRACE),
     INFEASIBLE_JOIN(DEBUG),
     SPARSE_INDEX_NOT_ELIGIBLE(TRACE),
-    HJ_NO_EQUIJOIN_COLUMNS(DEBUG);
+    HJ_NO_EQUIJOIN_COLUMNS(DEBUG),
+    REWIND_JOINORDER(TRACE),
+    PULL_OPTIMIZAL(TRACE);
```

Comment:
Should this be PULL_OPTIMIZABLE ?

Reply:
Thanks @msirek for catching the typo. I've fixed it.